tests/Tests/test.robot

```diff
@@ -26,9 +26,10 @@ Can Spawn Notebook
    # We need to skip this testcase if the user has an existing pod
    ${spawner_visible} =  JupyterHub Spawner Is Visible
    Skip If  ${spawner_visible}!=True  The user has an existing notebook pod running
+   Select Notebook Image  s2i-generic-data-science-notebook
    Select Notebook Image  s2i-minimal-notebook
-   Select Notebook Image  s2i-scipy-notebook
-   Select Notebook Image  s2i-tensorflow-notebook
+   #Select Notebook Image  s2i-scipy-notebook
+   #Select Notebook Image  s2i-tensorflow-notebook
```

Comment:
You can delete these commented out lines. That was original code that was included as a stress test of the automation and actual webui.

Reply:
Done in new commit